### ISSUE TITLE
! in this line throws a compile error in typescript.

### ISSUE THREAD

**Issue by akashdathan** [Kononnable/typeorm-model-generator#163]
https://github.com/Kononnable/typeorm-model-generator/blob/b70aba10c8baa4e373e86d3e4fdd2ab464f351ae/src/NamingStrategy.ts#L15

**Comment by Kononnable** [comment]
Hi there,
what error does it throw, why do you need to compile it - you've forked the project and want to make changes and compile it, copied sources to your project and want to include as part of your project(it's compiled with your code)?

**Comment by akashdathan** [comment]
Hi, sorry for the late reply. 
As mentioned in the docs, I created a naming strategy file in my code base and pointed the path to the new file. The new naming strategy file consists the copy of your NamingStrategy.ts file with few modifications by me. Since the file I created was also in typescript, I had to compile it down to js so that it is taken as a param. 
So the compilation is done just for the new NamingStrategy.ts which is a copy of yours. and this compilation throws error at line 15.

**Comment by Kononnable** [comment]
What is the error, what is your typescript version?

**Comment by akashdathan** [comment]
I was using an older version (Version 1.5.3). I have updated the version to 3.5.2 and it throws the following errors on compile.
`naming_strategy.ts:8:37 - error TS2339: Property 'find' does not exist on type 'any[]'.

8         const ownerEntity = dbModel.find(
                                      ~~~~

naming_strategy.ts:14:38 - error TS2339: Property 'endsWith' does not exist on type 'string'.

14             columnName.toLowerCase().endsWith("id") &&
                                        ~~~~~~~~

naming_strategy.ts:15:39 - error TS2339: Property 'endsWith' does not exist on type 'string'.

15             !columnName.toLowerCase().endsWith("guid")
                                         ~~~~~~~~


Found 3 errors.`

Ps : I have made some tweaks and made it work. But I don't think this approach is the right one (copying NamingStrategy file from your codebase to mine).
